README.md

```diff
@@ -49,11 +50,11 @@ sudo ldconfig
 To build and install the thrift compiler, run:
 
 ```
-wget -nv http://archive.apache.org/dist/thrift/0.7.0/thrift-0.7.0.tar.gz
-tar xzf thrift-0.7.0.tar.gz
-cd thrift-0.7.0
+wget -nv http://archive.apache.org/dist/thrift/0.9.3/thrift-0.9.3.tar.gz
+tar xzf thrift-0.9.3.tar.gz
```

Comment:
Parquet master CI is running against Thrift 0.12.0: https://github.com/apache/parquet-mr/commit/9d1006f853c99387834b10fc37e2c3548b2034d9

Reply:
This PR is targeted to branch `parquet-1.10.x` and not `master` so `0.9.3` is correct. We'll need another PR with `0.12.0` to master. @dongjoon-hyun, would you like to do another PR for `master` as well?

Reply:
Good catch, thanks!